maro/rl/training/training_manager.py

```diff
@@ -51,9 +51,7 @@ def __init__(
 
         self._agent2trainer: Dict[Any, str] = {}
         for agent_name, policy_name in self._agent2policy.items():
-            trainer_name = extract_trainer_name(policy_name)
-            if trainer_name in self._trainer_dict:
-                self._agent2trainer[agent_name] = trainer_name
+            self._agent2trainer[agent_name] = extract_trainer_name(policy_name)
```

Comment:
I think maybe we can add a check here to ensure every policy has exactly one corresponding trainer?

Reply:
Added.